react/Textarea/Textarea.demo.js

```diff
@@ -71,7 +71,7 @@ export default {
           })
         },
         {
-          label: 'Show Count',
+          label: 'Count',
```

Comment:
Did you mean to change this?

Reply:
I'm assuming you did this to make everything fit on one row on the demo page? Could we try adding `flex-wrap: wrap` to the container that wraps the options on the demo page instead?